StringList/Proj1_2.0/StringList.cpp

```diff
@@ -0,0 +1,310 @@
+﻿#define _CRT_SECURE_NO_WARNINGS
+#include"StringList.h"
+#include<stdlib.h>
+#include<string.h>
+#include<stdio.h>
+#include"node.h"
+
+void setNext(char** ptr, char** adr)//set pointer to the next element
```

Comment:
It's one-line function. Make sense to use "inline" keyword or create macros (like you made with getData).